src/name_bazaar/ui/core.cljs

```diff
@@ -51,6 +51,7 @@
                                      :events #{:district0x/set-active-page}
                                      :dispatch-to [:active-page-changed]}
                     :dispatch-n [[:setup-update-now-interval]
+                                 [:setup-address-reload-interval]
```

Comment:
Events should be as a part of district0x library and setup being fired in `:district0x/initialize`.